src/trips/tripsgo/dataAccess.go

```diff
@@ -15,13 +17,38 @@ func getEnv(key, fallback string) string {
 	return fallback
 }
 
+func writeConfigMessage(name string, value string, srcType string, src string) {
+	msg := ""
+	if value == "" {
+		msg = "no "
+	}
+	fmt.Printf("Config '%s' has %svalue set from %s '%s'.\n", name, msg, srcType, src)
```

Comment:
Will this print the value of the env to stdout?

Reply:
%svalue => %s value

Reply:
This code doesn't print out the value. It prints out whether it was able to obtain a value from that source.
'%svalue' is correct since %s can be '' or 'no ' - so it prints out 'value' or 'no value'